.gitignore

```diff
@@ -1,2 +1,4 @@
 src/_site
 src/Gemfile.lock
+respell-schema.py
+.vscode/
```

Comment:
what's in your .vscode/? some of those things we might want in the repo

keeping the respell-schema.py might be useful as an example? maybe put the date in front of the filename?

Reply:
respell-schema.py just iterates over Series -> Game & Randomizer, calling a function on each. Expectation is new requirements will require new code anyway :P 

.vscode/ contains workspace configs specific to a YAML schema validation plugin - I don't think they belong here, but extension/config could be linked in the contribution section of the README if it's really wanted

Reply:
I mean yea but an example code might help people? idk, I mean I've done transformations before and they always end up really simple and specific but it could help someone?

.vscode/ can contain an extensions.json to suggest that extension, and the settings for it, I think it's a good idea to include and I believe that's a big reason why VSCode supports these files in the workspace/repo

like for DXRando we have extensions.json:

```
{
	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp

	// List of extensions which should be recommended for users of this workspace.
	"recommendations": [
		"EditorConfig.EditorConfig", "EricBlade.vscode-unrealscript"
	],
	// List of extensions recommended by VS Code that should not be recommended for users of this workspace.
	"unwantedRecommendations": [
        "EliotVU.uc"
	]
}
```

where `EliotVU.uc` is an old extension for UnrealScript that doesn't work, and we recommend the other extension for UnrealScript instead, it helps people getting set up